docs/source/how_to_guides/optimization/how_to_specify_algorithm_and_algo_options.rst

```diff
@@ -679,18 +679,24 @@ noisy criterion functions.
       Function argument to find_affine_points() for the case where input array
       *model_improving_points* is zero.
     - **c2** (int): Treshold for accepting the norm of our current x candidate.
-      Equal to 10 by default. Argument to find_affine_points() in case
+      Equal to 10 by default. Argument to *find_affine_points()* in case
       the input array *model_improving_points* is not zero.
     - **trustregion_subproblem_solver** (str): Solver to use for the trust-region
       subproblem. Two internal solvers are supported:
       - "bntr": Bounded Newton Trust-Region (default, supports bound constraints)
       - "gqtpar": (does not support bound constraints)
+    - **trustregion_subproblem_conjugate_gradient_step** (str): Routine for computing
+      the conjugate gradient step, when the subsolver "bntr" is used.
+      Available conjugate gradient routines are:
+      - "standard"
+      - "steihaug-toint"
```

Comment:
I would prefer an underscore, i.e. `"steihaug_toint"`.

Reply:
It should also be part of the `trustregion_subproblem_options` which I would probably rename to `trustregion_subsolver_options`. Inside that dict in can just be called `conjugate_gradient_step`.

Reply:
Will do! I prefer to call it ``conjugate_gradient_routine``, because of a name conflict inside ``minimize_bntr()``. There,``conjugate_gradient_step`` is the actual cg step taken.

Reply:
Would `method` work? Routine sounds so much like Fortran.

Reply:
Sure! Another candidate would be ``algo``. But I like ``method``, too.